.ci/aarch64/kubernetes/init.sh

```diff
@@ -8,5 +8,12 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-k8s_version=$(kubectl version | base64 | tr -d '\n')
-network_plugin_config="https://cloud.weave.works/k8s/net?k8s-version=${k8s_version}"
+network_plugin_config_file="${SCRIPT_PATH}/../../.ci/${arch}/kubernetes/kube-flannel.yml"
+
+# original flannel has limitation and request for memory, it may cause OOM on AArch64
+sudo -E curl -fsL "https://raw.githubusercontent.com/coreos/flannel/bc79dd1505b0c8681ece4de4c0d86c5cd2643275/Documentation/kube-flannel.yml" -o $network_plugin_config_file
```

Comment:
- It looks like this (network) command doesn't need to run as `root` as the path is inside the repo?
- If we have to hard-code this commit id, please could you extract it and then re-add into the URL for further clarity. Something like:
  ```sh
  # original flannel has limitation and request for memory, it may cause OOM on AArch64
  flannel_version=bc79dd1505b0c8681ece4de4c0d86c5cd2643275

  flannel_file="kube-flannel.yml"
  flannel_url="https://raw.githubusercontent.com/coreos/flannel/${flannel_version}/Documentation/${flannel_file}
 
  curl -fsL -o "$network_plugin_config_file" "$flannel_url"
  ```

Reply:
Hi~ @jodh-intel  I have been using the same url link with the `amd64`, you could see from [here](https://github.com/kata-containers/tests/blob/master/integration/kubernetes/init.sh#L78). 
The configuration file has covered all archs, including `amd64`, `arm`, `arm64`,etc.
Getting inspiration from you, maybe I should move all your suggestion into common code, and just re-using the `flannel_url` in my arch-specific file. 😍

Reply:
Oh! Sorry @Pennyzct! leave any further rework for a follow-on PR.